index.html

```diff
@@ -18,15 +18,20 @@
     <header class="header backgrnd-clr">
       <nav class="navbar">
         <a href="index.html" class="logo">Welcome</a>
-        <img src="images/icons/Union.png" alt="hamburger-menu" />
+        <img class="hamburger-menu" src="images/icons/Union.png" alt="hamburger-menu" />
+        <ul class="navlist">
+          <li class="navitem"><a class="navlinks" href="#work">Portfolio</a></li>
+          <li class="navitem"><a class="navlinks" href="#about">About</a></li>
+          <li class="navitem"><a class="navlinks" href="#contact">contact</a></li>
+        </ul>
       </nav>
     </header>
     <main class="main-content">
       <!--Headline Section-->
       <section class="head-line backgrnd-clr">
         <h2 class="greeting greeting-text headline-box">
-          Hey there. I’m <br />
-          Mariane
+          Hey there. I’m <br class="break" />
+          <span class="ma">Mariane</span>  <span class="Al">Alex</span>
           <span class="text-color">I’m a software developer</span>
```

Comment:
- [ ] please kindly ensure that The Title heading is followed by 2 rows with 3 projects each (6 projects in total). Ensure they stay 3 projects in 2 rows on bigger screens.
![Screenshot (322)_LI](https://user-images.githubusercontent.com/65298048/186501535-3782cbee-5481-4040-ba92-49bc962f9e1b.jpg)